### DISCUSSION TITLE
Contributions

### DISCUSSION THREAD

**Discussion by morgverd** [lardbit/nefarious#188] [Q&A]
Hello. I wanted to start by saying how much I love this project. It’s simplistic in all the right ways while still giving all the functionality a user would need.

I’ve been working slowly on implementing OAuth SSO using OpenID and was wondering if the project takes pull requests, and if so is there any strict formatting requirements?

Thanks, morgverd.

**Comment by lardbit** [comment]
Hey @MorgVerd,

Thanks for the good words! I gladly accept contributions.  You'd be the first code contributor, and I haven't yet nailed down strict formatting requirements, or enforced them in anyway.  However, I've always tried to stay inline with [pep8](https://github.com/lardbit/nefarious/discussions/188) for python.  As for typescript/javascript, I don't have as much familiarity with style guides, but i try and follow suit with the style used by [Angular](http://angular.io/) (the frontend library nefarious uses).

I'd love some contributions, so don't get too caught up worrying about styles at first.  We can sort that out down the road.

Let me know if I can help get you up and running in any way.
